### ISSUE TITLE
Multiline functions indent

### ISSUE THREAD

**Issue by czjvic** [eslint/eslint#3197]
Is there some rule, which support this indent? 

Wrong (I need error/warning for this situation):
```javascript
myFunction(
    'aaaaaaaaaaa',
    'aaaaaaaaaaa',
    );
```

Expected (close brace at the same column as function):
```javascript
myFunction(
    'aaaaaaaaaaa',
    'aaaaaaaaaaa',
);
```

I tried every configuration param in 1.0.0 RC3 version but I didn't find it.

**Comment by eslintbot** [comment]
Thanks for the issue! We get a lot of issues, so this message is automatically posted to each one to help you check that you've included all of the information we need to help you.

Reporting a bug? Please be sure to include:

1. The version of ESLint you are using (run `eslint -v`)
1. The source code that caused the problem
1. The configuration you're using (for the rule or your entire config file)
1. The actual ESLint output complete with line numbers

Requesting a new rule? Please be sure to include:

1. The use case for the rule - what is it trying to prevent or flag?
1. Whether the rule is trying to prevent an error or is purely stylistic
1. Why you believe this rule is generic enough to be included

Requesting a feature? Please be sure to include:

1. The problem you want to solve (don't mention the solution)
1. Your take on the correct solution to problem

Including this information in your issue helps us to triage it and get you a response as quickly as possible.

Thanks!

**Comment by gyandeeps** [comment]
Duplicate https://github.com/eslint/eslint/issues/1801
I will close this issue, please follow https://github.com/eslint/eslint/issues/1801 for updates.
Thanks
